### PR TITLE
enable CDN inside the editor

### PR DESCRIPTION
**Pull request by javisantana**

### CODE REVIEW

**Comment by javisantana** [comment]
restest this please

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2692/))

**Comment by javisantana** [comment]
cc @alonsogarciapablo review

File: lib/assets/javascripts/cartodb/table/mapview.js [hunk 1]
```diff
@@ -162,7 +162,7 @@ function GrouperLayerMapView(mapViewClass) {
               tiler_domain: cdb.config.get('tiler_domain'),
               tiler_port: cdb.config.get('tiler_port'),
               tiler_protocol: cdb.config.get('tiler_protocol'),
-              no_cdn: true,
+              no_cdn: false,
```
Comment (alonsogarciapablo) [inline lib/assets/javascripts/cartodb/table/mapview.js]: The only thing I'm missing is a test for this guy, but I'm sure you're aware of that.

**Comment by alonsogarciapablo** [comment]
:+1:

**Comment by saleiva** [comment]
bump!

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2952/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2953/))

**Pull request merged**